### ISSUE TITLE
Feature Request: privacy group editing, select all contacts of an group for further steps

### ISSUE THREAD

**Issue by tschlotfeldt** [friendica/friendica#1569]
Dealing with a lot of contacts can be complicated. Sometimes it would be nice to have an easy to use tool to filter contacts by one ore more groups and to select one or many contact of a group in an easy way.

In a first step it would be nice to have a kind of mass contact group editing, e.g. mark all members of a group  and let the marked contact

  - move them to another group,
 - create a new group for them or
 - let them also be member of another group.

**Comment by annando** [comment]
I would like to have features like selecting all contacts from a network and copying them to a group and such stuff.

This is much frontend stuff - I cannot do frontend stuff. But I will see if I can create the needed background tasks.

**Comment by anubis2814** [comment]
Duplicating as well as merging groups would also be very useful.

**Comment by AlfredSK** [comment]
...milestone 3.6.1 :-)

**Comment by anubis2814** [comment]
Would it be possible to share a group with someone else?

**Comment by MrPetovan** [comment]
Do you mean sharing a list of follows or show who's on one of your groups?

**Comment by anubis2814** [comment]
Share the entire group to someone else so they can easily see the feed like I do, maybe even an auto follow option for them all.

**Comment by MrPetovan** [comment]
So show who's on one of your groups through a filtered stream. It should be possible, albeit only their public posts.

**Comment by anubis2814** [comment]
Yes that would make sense. would require permissions from the individual people followed.  There are several people who would probably be more easily swayed to join if I could set them up with some of my groups because otherwise it requires people to hunt them down individually and create them.  Mine are now working like groups on MEWE which is really nice and having those streams ready made by friends would make the place feel more inviting.

**Comment by MrPetovan** [comment]
Why would you need permission from the individual people followed to group them in a stream of public posts?

**Comment by anubis2814** [comment]
Stream version of a personalized mix tape.

**Comment by anubis2814** [comment]
No if the person you shared the stream with wanted to see private posts from people in the stream it would require getting permission to follow them.  Unless it was the type that allowed anyone to join (soapbox)

**Comment by MrPetovan** [comment]
Yes, I see what you mean.

**Comment by anubis2814** [comment]
Too many parties involved and if I'm not specific I completely lose ya lol
